.gitignore

```diff
@@ -1,2 +1,4 @@
 *.egg-info
 *.pyc
+.idea/
```

Comment:
please don't add environment-specific things to project gitignores. you can use a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) to ignore files from your local environment across all projects and then we don't need to have everyone's local setup accounted for in each project.